ui/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala

```diff
@@ -90,6 +90,13 @@ class ProcessesResources(val processRepository: FetchingProcessRepository,
             }
           }
         } ~ path("processesDetails") {
+          post {
+            entity(as[List[String]]) { namesToFetch =>
```

Comment:
I wonder if it would be more "RESTful" if we did it with GET and passed ids as query string?